### PR TITLE
Fixing long anchors - right spacing

### PR DESCRIPTION
**Pull request by marina-marina**
`['anna luise', 'jones'] => 'anna-luise-jones'`

### CODE REVIEW

File: src/components/SpeakerBlock/SpeakerBlock.js [hunk 1]
```diff
@@ -3,45 +3,52 @@ import './speaker-block.scss';
...
 
+  static defaultProps = {
+    data: {},
+  }
+
   render() {
-    const { data } = this.props;
+    const {
+      data: { first_name, last_name, image_src, social_icons, position, company, description }
+    } = this.props;
```
Comment (tyv) [inline src/components/SpeakerBlock/SpeakerBlock.js]:
```js
const { 
  first_name,
  last_name,
  image_src,
  social_icons,
  position,
  company,
  description 
} = this.props.data
```

File: src/components/SpeakerBlock/SpeakerBlock.js [hunk 2]
```diff
@@ -3,45 +3,52 @@ import './speaker-block.scss';
...
     const imageExtension = isChrome() ? 'webp' : 'jpg';
 
     return (
       <div className="speaker-block container container-fluid">
         <div className="speaker-block-cols">
           <a
             href=""
-            name={`${data.first_name}-${data.last_name}`}
+            name={getAnchorName([first_name, last_name])}
```
Comment (tyv) [inline src/components/SpeakerBlock/SpeakerBlock.js]: why array? just `getAnchorName(first_name, last_name)`

File: src/utils/common.js [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
 }
+
+export function replaceSpaces(str) {
+  return str.replace(/ /, '-');
+}
+
+export function getAnchorName(arr) {
+  // ['anna luise', 'jones'] => `anna-luise-jones`
+  return arr.map(el => replaceSpaces(el)).join('-');
+}
```
Comment (tyv) [inline src/utils/common.js]: `replaceSpaces` isn't used anywhere except one place, right? first no need to export and second no need for that function.

and no need in `getAnchorName`

File: src/components/SpeakerBlock/SpeakerBlock.js (hunk 2 above)
Comment (marina-marina) [reply src/components/SpeakerBlock/SpeakerBlock.js]: Because I wanted to make it reusable and extensible.

Comment (tyv) [reply src/components/SpeakerBlock/SpeakerBlock.js]:
```js
name={`${first_name}-${last_name}`.replace(/\s/g, '-')}
```

Comment (marina-marina) [reply src/components/SpeakerBlock/SpeakerBlock.js]: thanks! the shortest and the best

**Pull request merged**